packages/xrpl/src/models/transactions/payment.ts

```diff
@@ -166,6 +166,24 @@ export interface PaymentMetadata extends TransactionMetadataBase {
 export function validatePayment(tx: Record<string, unknown>): void {
   validateBaseTransaction(tx)
 
+  if (tx.Amount === undefined) {
```

Comment:
```suggestion
  if (tx.Amount == null) {
```
It's better to use value comparison with null `== null` since it will cover cases for both `undefined` and `null`.